substrate/frame/beefy/src/lib.rs

```diff
@@ -133,6 +133,22 @@ pub mod pallet {
 	pub type NextAuthorities<T: Config> =
 		StorageValue<_, BoundedVec<T::BeefyId, T::MaxAuthorities>, ValueQuery>;
 
+	/// publicly verifiable shares for the current round (a resharing)
+	#[pallet::storage]
+	pub type Shares<T: Config> = 
+		StorageValue<_, BoundedVec<BoundedVec<u8, ConstU32<1024>>, T::MaxAuthorities>, ValueQuery>;
```

Comment:
Is `BoundedVec<u8, ConstU32<1024>>` a share? Would it make sense to create a type for it?